common/stage.py

```diff
@@ -2088,6 +2299,9 @@ def generate_map_offset_table( self ):
 		'''
 		Generates list of offsets for each of the interconnect ids
 		'''
+		print("{0} This functionality is handled by the preprocessor".format(ERROR))
+		# FIXME Should this me removed entirely?
```

Comment:
Could not really figure out what was the purpose of this step here/how it was supposed to work. As far as I know, it is not being superseded by the preprocessor stage.